features/achordion.c

```diff
@@ -135,7 +135,7 @@ bool process_achordion(uint16_t keycode, keyrecord_t* record) {
     }
 
 #ifdef ACHORDION_STREAK
-    streak_timer = (timer_read() + achordion_streak_timeout(keycode)) | 1;
+    streak_timer = record->event.time | 1;
```

Comment:
This line is reached when in STATE_RELEASED and a non-tap-hold key is pressed. Is that intended? To measure time starting from the tap-hold key press, this line should move up a few lines within the `if (is_tap_hold...` if-block.

Reply:
Yep. I'm trying to time between 2 taps that sandwich a hold.